stdlib/internals.ncl

```diff
@@ -3,50 +3,50 @@
   # valid starting character for an identifier.
 
   # Contract implementations
-  "$dyn" = fun l t => t,
+  "$dyn" = fun _label value => value,
 
-  "$num" = fun l t => if %typeof% t == `Number then t else %blame% l,
+  "$num" = fun label value => if %typeof% value == `Number then value else %blame% label,
 
-  "$bool" = fun l t => if %typeof% t == `Bool then t else %blame% l,
+  "$bool" = fun label value => if %typeof% value == `Bool then value else %blame% label,
 
-  "$string" = fun l t => if %typeof% t == `String then t else %blame% l,
+  "$string" = fun label value => if %typeof% value == `String then value else %blame% label,
 
-  "$fail" = fun l t => %blame% l,
+  "$fail" = fun label _value => %blame% label,
 
-  "$array" = fun elt l t =>
-    if %typeof% t == `Array then
-        %array_lazy_assume% (%go_array% l) t elt
+  "$array" = fun element-contract label value =>
+    if %typeof% value == `Array then
+        %array_lazy_assume% (%go_array% label) value element-contract
     else
-        %blame% l,
+        %blame% label,
 
-  "$func" = fun s t l e =>
-      if %typeof% e == `Function then
-          (fun x => %assume% t (%go_codom% l) (e (%assume% s (%chng_pol% (%go_dom% l)) x)))
+  "$func" = fun domain codomain label value =>
+      if %typeof% value == `Function then
+          (fun x => %assume% codomain (%go_codom% label) (value (%assume% domain (%chng_pol% (%go_dom% label)) x)))
       else
-          %blame% l,
+          %blame% label,
 
-  "$forall_var" = fun sy pol l t =>
-      let lPol = %polarity% l in
-      if pol == lPol then
-          %unseal% sy t (%blame% l)
+  "$forall_var" = fun sealing-key polarity label value =>
```

Comment:
I didn't know Nickel variable names could have a `-` in them 😅

Reply:
Not sure if that fits the coding style of stdlib right now, but I felt like kebab 😛

Reply:
@ebresafegaga Yup :slightly_smiling_face: it was made to accommodate for potential target configurations that use  options with this style. So it's typically rather for exported record fields (whose name is not chosen by you), but all identifiers have the same syntax, so you can indeed use a dash inside a function variable or a let-binding.